<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="RevFlow R6 Login Test"
           tests="2"
           failures="2"
           errors="0"
           skipped="0"
           time="0.025">
  <properties />
            <testcase name="LoginTest" time="0.002" classname="com.revflow.testcase.TestCase001">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  com.google.common.base.Preconditions.checkNotNull(Preconditions.java:882)
  org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:104)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:71)
  org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)
  com.revflow.page.Login.verifyPageLoaded(Login.java:161)
  com.revflow.testcase.TestCase001.LoginTest(TestCase001.java:34)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="LogOutTest" time="0.023" classname="com.revflow.testcase.TestCase001">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
  org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
  com.sun.proxy.$Proxy9.click(Unknown Source)
  com.revflow.page.HomePage.clickAdminLink(HomePage.java:81)
  com.revflow.testcase.TestCase001.LogOutTest(TestCase001.java:48)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
